app/routers/api/encryption.py

```
@@ -77,3 +162,241 @@ def delete_key(
+                            name = storage_reference.rstrip("/").split("/")[-1]
+                            relative_path = Path(name)
+
+                        # Upload to backend
+                        ul_success, ul_error, new_reference, _ = backend.freeze_file(
+                            source_path=enc_path,
+                            relative_path=relative_path,
+                            location=file.storage_location,
+                            operation_mode=OperationType.COPY,
+                        )
+                        if not ul_success:
+                            logger.error(f"Failed to upload migrated file {file.id}: {ul_error}")
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)